co2eq/meal.js

```diff
@@ -117,14 +121,15 @@ const CARBON_INTENSITY = { // kgCO2eq / kg
   'Eggplants': 1.35, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   // staples
   'Rice': 2.55, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
-  'Nuts': 2.3, // http://www.greeneatz.com/foods-carbon-footprint.html
+  'Nuts': 2.3, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
   'Pasta': 1.2, // From https://naturerhverv.dk/fileadmin/user_upload/NaturErhverv/Filer/Tvaergaaende/Foedevarernes_klimaaftryk_tabel_1.pdf
   'Rye bread': 0.93, // From http://web.agrsci.dk/djfpublikation/djfpdf/DCArapport158.pdf
   'White bread': 1.054, // From http://web.agrsci.dk/djfpublikation/djfpdf/DCArapport158.pdf
   'Morning cereals': 0.864, // From http://web.agrsci.dk/djfpublikation/djfpdf/DCArapport158.pdf
   'Oats': 0.38, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
   'Green beans': 0.31, // https://www.sciencedirect.com/science/article/pii/S0959652616303584
-  'Beans/tofu': 2.0, // From http://www.greeneatz.com/foods-carbon-footprint.html
+  'Dry beans': 2.0, // http://static.ewg.org/reports/2011/meateaters/pdf/methodology_ewg_meat_eaters_guide_to_health_and_climate_2011.pdf page 23
```

Comment:
Let's not replace existing items as this would currently break past meals, feel free to add the two additionally